### PR TITLE
front: rs-editor: add base power class field

### LINKED ISSUE #5175: front : be able to enter the base power class of an RS in "general characteristics"
## Description

When editing/creating a rolling stock, user need to be able to set a default power class which will be associated with the "Non Specified" power restriction code. 
The user needs also to see the default power class when selecting a rolling stock.

## AC

- [ ] A new field should be displayed next to "gauge" when looking at a rolling stock details
- [ ] A new field should be added under the "gauge" field in the general characteristics tab
- [ ] The field should be called "Power Class" / "Classe de puissance" 
- [ ] The field should be a string and not required

### PR DESCRIPTION
**Pull request by SharglutDev**
close #5175

### CODE REVIEW

**Review by Akctarus — APPROVED** [review]
Tested in local with different values and it works well, LGTM

**Comment by kmer2016** [comment]
There are some fields in `RollingStockParametersValues` which is not used in Front

**Review by kmer2016 — APPROVED** [review]

File: front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx [hunk 1]
```diff
@@ -202,11 +203,16 @@ const RollingStockEditorParameterFormColumn = ({
...
+              property.title !== 'basePowerClass'
+                ? (rollingStockValues[property.title] as number)
+                : (rollingStockValues[property.title] as string)
+            }
             onChange={(e) =>
               setRollingStockValues({
                 ...rollingStockValues,
-                [property.title]: parseFloat(e.target.value),
+                [property.title]:
+                  property.title !== 'basePowerClass' ? parseFloat(e.target.value) : e.target.value,
```
Comment (Math-R) [inline front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx]: you could use the + operator instead of parseFloat

Comment (SharglutDev) [reply front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx]: done

**Review by Math-R — APPROVED** [review]

**Pull request merged**